docs/troubleshooting/kubernetes.rst

```diff
@@ -1,13 +1,223 @@
 Troubleshoot Your Kubernetes Deployment
 =======================================
 
-BIG-IP Controller
------------------
+.. toctree::
+   :maxdepth: 1
+
+How to get help
+---------------
+
+If the issue you're experiencing isn't covered here, try one of the following options:
+
+- `Contact F5 Support`_ (valid support contract required).
+- `Report a bug <https://github.com/F5Networks/k8s-bigip-ctlr/issues>`_ in the k8s-bigip-ctlr GitHub repo.
+- `Ask a question <https://f5cloudsolutions.slack.com>`_ in the #cc-kubernetes channel in the F5 Cloud Solutions Slack team.
+
+
+General Kubernetes and OpenShift troubleshooting
+------------------------------------------------
+
+The following troubleshooting docs may help with Kubernetes- or OpenShift-specific issues.
+
+- `Kubernetes: Troubleshoot Applications <https://kubernetes.io/docs/tasks/debug-application-cluster/debug-application/>`_
+- `Kubernetes: Troubleshoot Clusters <https://kubernetes.io/docs/tasks/debug-application-cluster/debug-cluster/>`_
+- `OpenShift: Troubleshooting OpenShift SDN <https://docs.openshift.org/1.5/admin_guide/sdn_troubleshooting.html>`_
+
+.. _k8s-bigip-ctlr troubleshoot:
+
+BIG-IP Controller for Kubernetes/OpenShift
+------------------------------------------
+
+.. _common troubleshoot:
+
+Common Issues
+`````````````
+
+How do I check the settings of my |kctlr| using the command line?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can use `kubectl`_ or `oc`_ commands to check the |kctlr| configurations using the command line.
+
+.. code-block:: console
+
+   kubectl [|oc] get pod -o yaml [--namespace=kube-system]          \\ Returns the Pod's YAML settings
+   kubectl [|oc] describe pod myBigIpCtlr [--namespace=kube-system] \\ Returns an information dump about the Pod you can use to troubleshoot specific issues
+
+-----------------------------------------
+
+What happened to my BIG-IP configuration changes?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you make changes to objects in the partition managed by the |kctlr| -- whether via configuration sync or manually -- **the Controller will overwrite them**. By design, the |kctlr| keeps the BIG-IP system  in sync with what it knows to be the desired configuration. For this reason, F5 does not recommend making any manual changes to objects in the partition(s) managed by the |kctlr|.
```

Comment:
looks like an extra space between "BIG-IP system" and "in sync"

Reply:
fixed